swan/files/swan_config.py

```diff
@@ -175,21 +175,20 @@ def swan_pod_hook(spawner, pod):
 
 # Manage EOS access
 if get_config("custom.eos.enabled", False):
-    # Access via bind-mount from the host
+    c.SwanKubeSpawner.volumes.append(
+        V1Volume(
+            name='eos',
+            persistent_volume_claim=V1PersistentVolumeClaimVolumeSource(
+                claim_name='eos'
+            )
+        )
+    )
     c.SwanKubeSpawner.volume_mounts.append(
         V1VolumeMount(
             name='eos',
             mount_path='/eos',
             mount_propagation='HostToContainer'
-        ),
-    )
-    c.SwanKubeSpawner.volumes.append(
-        V1Volume(
-            name='eos',
-            host_path=V1HostPathVolumeSource(
-                path=get_config("custom.eos.automountHostPath", "/var/eos")
```

Comment:
`custom.eos.automountHostPath` is not needed anymore, then? We can perhaps get rid of the anchor and just set it as a config argument for the eosxd-csi chart?

Reply:
I removed it yes